components/storage/ComparisonTable.vue

```diff
@@ -59,6 +52,15 @@ export default {
       required: true,
     },
   },
+  methods: {
+    humanize(str) {
```

Comment:
maybe move `urlize` and `humanize` to a `utils.js` file and import where needed - seems like they're used all over